hotel-management/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="./favicon-32x32.png" type="image/x-icon">
```

Comment:
Why not using `.ico` extension?

Reply:
Now, I find that choosing .ico for the favicon is better due to its browser support. Despite .png's customization ease, it doesn't suit my needs.